spec/worker_spec.js

```diff
@@ -0,0 +1,30 @@
+var noop = require("lodash.noop");
+
+describe("worker", function() {
+  it("should handle message correctly", function() {
+    var fakeJasmine = jasmine.createSpyObj("fakeJasmine", ["onComplete"]),
+        fakeNewJasmine = jasmine.createSpy("fakeNewJasmine").and.returnValue(fakeJasmine),
+        fakeRunJasmine = jasmine.createSpy("fakeRunJasmine"),
+        fakeRequire = {
+          "./new": fakeNewJasmine,
+          "./run": fakeRunJasmine
+        },
+        env = {};
+    
+    spyOn(require("module"), "_load").and.callFake(function() {
```

Comment:
Again, we should not be spying on `module._load` but injecting it instead.

Reply:
See my first reply. let's use the first one to communicate